### DISCUSSION TITLE
How to determine QGIS version supported by particular QField/QFieldCloud release?

### DISCUSSION THREAD

**Discussion by w0pr** [opengisch/QField#5869] [Q&A]
I am trying to determine the maximum possible QGIS version that is usable with QField and QFieldCloud pipeline; for QFieldCloud this can be determined by looking at the Dockerfile: 
`ARG QGIS_VERSION=1:3.40.1+40noble`
https://github.com/opengisch/QFieldCloud/blob/5b402212ef98c1266832c4e654be8c35b4794bed/docker-qgis/Dockerfile#L32

However, I can not find this information (the deployed version) anywhere on qfield.cloud website, only when looking at the job logs:
![image](https://github.com/user-attachments/assets/fad3b23d-687e-4c44-a5ec-9b4ec44924c3)

Similarly for QField; where can I I find the information which features of a particular QGIS version are supported in terms of project files and so on?

**Comment by suricactus** [comment]
For QFieldCloud short answer is - currently you cannot. You can have a close approximation by looking at:
https://github.com/opengisch/QFieldCloud/blob/master/docker-qgis/Dockerfile#L32

This is what is already on the current realease or on the upcoming release. At the very moment, it is what will be on the next release in few weeks.

The plans are that in the future one can select one of the many QGIS versions for their project. Until we will consider showing this information somewhere in the UI.

**Reply by w0pr** [reply]
Thanks, that's as far as I got; what about QField itself?
I have a similar situation with QGIS Web Client (QWC2) where some features of the version used are not implemented (yet).

But I can't even find a hint in QField release notes which QGIS version it is supposed to be "pinned" at.

**Reply by suricactus** [reply]
Similar situation with QField, you can check this information in the source code: https://github.com/opengisch/QField/blob/master/vcpkg/ports/qgis/portfile.cmake#L1-L2 . There it is actually easier to track what version you are running and when the QGIS version has been changed.

**Reply by suricactus** [reply]
Oh, my bad, you can see it in the variables tab:

![image](https://github.com/user-attachments/assets/ccda3899-6b8c-4677-8826-3254fa371776)

**Reply by w0pr** [reply]
Oh dear, it is actually easy to find in QField. My bad, actually.
